serialzy/serializers/sklearn.py

```diff
@@ -0,0 +1,59 @@
+from os import PathLike
+from pathlib import Path
+from typing import BinaryIO, Tuple, Type, Any, Optional, Union
+
+import pickle
+from serialzy.serializers.base_model import (
+    ModelBaseSerializer,
+    serialize_to_file,
+    deserialize_from_file,
+    unpack_model_file
+)
+
+
+class SciKitLearnSerializer(ModelBaseSerializer):
+    def __init__(self):
+        super().__init__("sklearn", __name__, package="scikit-learn")
+
+    def unpack_model(self, source: BinaryIO, dest_dir: Union[str, PathLike]) -> PathLike:
+        model_path = Path(dest_dir) / "model.pickle"
+        unpack_model_file(source, model_path)
+        return model_path
+
+    def _serialize(self, obj: Any, dest: BinaryIO) -> None:
+        if not isinstance(obj, SciKitLearnSerializer._get_supported_types()):
```

Comment:
Nonono, self would be better in case of inheritance, I can't see why you changed it to explicit classname